w3-store.md

```diff
@@ -1,631 +1,870 @@
-# Storage and upload protocol
+# W3 Storage Protocol
 
 ![status:wip](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
```

Comment:
SHould we state that this is in deprecation stage?

Reply:
I don't think we should deprecate until we have alternative